homeassistant/components/generic/camera.py

```diff
@@ -84,6 +85,9 @@ def __init__(self, hass, device_info):
         self._still_image_url = device_info[CONF_STILL_IMAGE_URL]
         self._stream_source = device_info.get(CONF_STREAM_SOURCE)
         self._still_image_url.hass = hass
+        self._attr_unique_id = "_".join(
+            [DOMAIN, slugify(str(self._still_image_url.template))]
+        )
```

Comment:
This does not fulfill the unique ID requirements:

<https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements>

Reply:
:open_mouth: indeed...
So, I don't think there is a way to generate an unique id for this entity, according to this list :(

Reply:
Can we allow the user to define one from yaml (like mqtt) ?

Reply:
I remove this code